package.json

```diff
@@ -42,21 +42,21 @@
 		"configstore": "^6.0.0",
 		"import-lazy": "^4.0.0",
 		"is-in-ci": "^0.1.0",
-		"is-installed-globally": "^0.4.0",
+		"is-installed-globally": "^1.0.0",
 		"is-npm": "^6.0.0",
-		"latest-version": "^7.0.0",
+		"latest-version": "^9.0.0",
 		"pupa": "^3.1.0",
-		"semver": "^7.5.4",
+		"semver": "^7.6.2",
 		"semver-diff": "^4.0.0",
 		"xdg-basedir": "^5.1.0"
 	},
 	"devDependencies": {
 		"ava": "^5.3.1",
```

Comment:
AVA was not updated because:

```
test/update-notifier.js:28

   27:   setTimeout(() => {               
   28:     fs.unlinkSync(configstorePath);
   29:   }, 10_000);                      

  Error: ENOENT: no such file or directory, unlink '/home/runner/.config/configstore/update-notifier-update-notifier-tester.json'

  › Timeout._onTimeout (file://test/update-notifier.js:28:6)

  ✘ test/update-notifier.js exited with a non-zero exit code: 1
```

It can be updated separately